src/simulator/ackermann_model.h

```diff
@@ -0,0 +1,38 @@
+#include "f1tenth_simulator/AckermannCurvatureDriveMsg.h"
+#include <random>
+#include <string>
+
+#include "f1tenth_simulator/AckermannCurvatureDriveMsg.h"
+#include "ros/ros.h"
+#include "simulator/entity_base.h"
+
+#ifndef SRC_SIMULATOR_ACKERMANN_MODEL_H_
+#define SRC_SIMULATOR_ACKERMANN_MODEL_H_
+
+namespace ackermann {
+
+class AckermannModel: public EntityBase{
+ private:
+   float vel_;
+   f1tenth_simulator::AckermannCurvatureDriveMsg last_cmd_;
+   double t_last_cmd_;
+   std::default_random_engine rng_;
```

Comment:
Much more far ranging thought: will we ever need to fixed seed the RNG to fix the noise for experimentation?

Reply:
It's possible that someone may want this in the future, but I think we can probably cross that bridge when we get to it.